### PR TITLE
SW-5289 Species table row modifications

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2024-05-07 at 12 06 34 PM

Table changes:
Scientific name click goes to “participant project species view” (this view is not done yet)
Added Reject and approve button in each row.
Removed add species button.
Added “reject species” dialog when clicking reject button in row

### CODE REVIEW

File: src/components/SpeciesDeliverableTable/index.tsx [hunk 1]
```diff
@@ -78,20 +80,20 @@ const SpeciesDeliverableTable = (): JSX.Element => {
...
+            {!isAcceleratorRoute && (
+              <Button
+                icon='plus'
+                id='add-species-to-project'
+                label='Add Species to Project'
+                onClick={() => setOpenedAddSpeciesModal(true)}
+                priority='secondary'
+                size='medium'
+              />
+            )}
```
Comment (joshbuchea) [inline src/components/SpeciesDeliverableTable/index.tsx:92]: Is it safe to assume all accelerator users have permission to use this feature or are there any permission checks that should be made here?

File: src/services/ParticipantProjectSpeciesService.ts [hunk 2]
```diff
@@ -107,7 +107,7 @@ const update = async (
     const index = speciesList.findIndex((ps) => ps.id === participantProjectSpeciesId);
 
     if (index !== -1) {
-      speciesList[index] = participantProjectSpecies;
+      // speciesList[index] = participantProjectSpecies;
```
Comment (joshbuchea) [inline src/services/ParticipantProjectSpeciesService.ts:110]: Was this for testing? Does this change need to be included?

**Review by joshbuchea — APPROVED** [review]

File: src/components/SpeciesDeliverableTable/index.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/SpeciesDeliverableTable/index.tsx:92]: Hmm I'm not sure if all users can add species. We can add condition later if needed

File: src/services/ParticipantProjectSpeciesService.ts (hunk 2 above)
Comment (constanzauanini) [reply src/services/ParticipantProjectSpeciesService.ts:110]: It was for testing, anyways mocks are removed now I think

**Pull request merged**